aocr/util/data_gen.py

```diff
@@ -52,10 +64,12 @@ def __init__(self,
         :param img_width_range: only needed for training set
         :param word_len:
         :param epochs:
+        :param augment_data_prob: probability of applying data augmentation functions on the sample
```

Comment:
It's best to have the docstring order match the order of the arguments themselves
```suggestion
        :param augment_data_prob: probability of applying data augmentation functions on the sample
        :param epochs:
```